supergloo/examples/prometheus/prometheus-demo.yaml

```diff
@@ -0,0 +1,814 @@
+---
```

Comment:
So this file is pretty complicated with all the stuff it has in it. Is it all necessary for this tutorial?